fxdc/config.py

```
@@ -65,39 +153,72 @@ def __init__(self) -> None:
+
+        Returns:
+            IdentityDeco: Decorator to add a class to the config
+
+        Usage:
+            ```py
+            @Config.add_class(name="MyClass", from_data=my_from_data, to_data=my_to_data, typechecking=True)
+            class MyClass:
+                pass
+            ```
+        """
         ...
```

Ruff (0.12.2)

Unnecessary `...` literal

Remove unnecessary `...`

(PIE790)